src/main/webapp/app/applications/instance/instance.vue

```diff
@@ -19,7 +19,21 @@
                         <a :href="instance.uri" target="_blank">{{ instance.serviceId }}</a>
                     </td>
                     <td class="table-hover">
-                        <span class="badge badge-pill badge-light">{{ instance.instanceId }}</span>
+                        <span class="badge badge-ligth">{{ instance.instanceId }}</span><br/>
```

Comment:
```suggestion
                        <span class="badge badge-light">{{ instance.instanceId }}</span><br/>
```

Reply:
An example of instances page : 

![jhcc_screen_1](https://user-images.githubusercontent.com/23552990/80083257-b1cdf280-8555-11ea-9dec-1e888139fb15.png)

Reply:
something to do here, we lost a lot of space at the middle of the screen

Reply:
@sendilkumarn for the moment, table is enough for the need, this is why we choosed it.

@pascalgrimaud I spreaded badges between columns to reduce blank space like this :

![jhcc_example](https://user-images.githubusercontent.com/23552990/80193505-359ce300-8619-11ea-993d-415044ad16c6.png)